macOS-Generic/InstallHuntress-macOS-bash.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/bash
+#!/bin/zsh
```

Comment:
Should we change `bash` to `zsh` in the filename while we're here?

Reply:
Yup, good catch